packages/blocks/src/image-block/ledits/utils.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
```suggestion
```

Reply:
You should only disable some of the rules instead of disable whole eslint.